src/flows/CostCalculator/hooks.ts

```diff
@@ -158,20 +158,30 @@ export const defaultEstimationOptions = {
   includeCostBreakdowns: false,
 };
 
+type UseCostCalculatorParams = {
+  defaultRegion?: string;
+  estimationOptions: CostCalculatorEstimationOptions;
+};
+
 /**
  * Hook to use the cost calculator.
  */
 export const useCostCalculator = (
-  estimationOptions: CostCalculatorEstimationOptions = defaultEstimationOptions,
+  { defaultRegion, estimationOptions }: UseCostCalculatorParams = {
+    estimationOptions: defaultEstimationOptions,
+  },
 ) => {
   const [selectedRegion, setSelectedRegion] = useState<string>();
   const [selectedCountry, setSelectedCountry] =
     useState<CostCalculatorCountry>();
   const { data: countries } = useCostCalculatorCountries();
   const { data: currencies } = useCompanyCurrencies();
-  const { data: jsonSchemaRegionFields } = useRegionFields(
-    selectedRegion || selectedCountry?.value,
-  );
+
+  const jsonSchemaRegionSlug =
+    selectedRegion || selectedCountry?.value || defaultRegion;
```

Comment:
Assuming you'll follow my suggestion above, we can rollback this change

```suggestion
  const jsonSchemaRegionSlug =
    selectedRegion || selectedCountry?.value;
```

Reply:
done